TAO/tests/Oneway_Send_Timeouts/main.cpp

```diff
@@ -126,7 +140,8 @@ MyMain::run ()
 
 MyMain::~MyMain ()
 {
-  if (!shutdown_) {
+  if (! shutdown_)
```

Comment:
We normally don't use a space between ! and shutdown_

Reply:
I found it very often

Reply:
ok, done